konrad/atmosphere.py

```diff
@@ -257,6 +258,24 @@ def refine_plev(self, pgrid, **kwargs):
 
         return new_atmosphere
 
+    def copy_atmosphere(self):
```

Comment:
It is common to name this kind of method simply ``copy`` as it is clear that ``Atmospgere.copy()`` copies the atmosphere 😉

Reply:
Is that a good idea, given that I import ``copy`` from ``copy``? Maybe it's fine, but it felt a bit unsafe to me...

Reply:
It should be fine as the name scope is clear as one is a stand-alone function ``copy`` and one is a method of the ``Atmosphere`` class (like ``numpy.mean`` and ``numpy.array.mean``).